recipes-core/data-partition/files/postinst

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+deb-systemd-helper enable data.mount || true
+
+echo "Updatind initramfs for containing fsck"
```

Comment:
This would better be a separate Pull Request, this one here is as the title suggests about adding SWUpdate.